Lib/test/test_concurrent_futures/test_interpreter_pool.py

```diff
@@ -282,6 +282,15 @@ def test_idle_thread_reuse(self):
 
 class AsyncioTest(InterpretersMixin, testasyncio_utils.TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        policy = support.maybe_get_event_loop_policy()
+        assert policy is None, policy
+
+    @classmethod
+    def tearDownClass(cls):
+        asyncio.set_event_loop_policy(None)
+
```

Comment:
It might be worth adding a comment as to what this is doing--it's not totally clear to me. Is it supposed to restore `policy` upon teardown?

Reply:
done